tests/migrations/test_operations.py

```diff
@@ -3626,6 +3626,26 @@ def test_rename_index_unknown_unnamed_index(self):
             with self.assertRaisesMessage(ValueError, msg):
                 operation.database_forwards(app_label, editor, project_state, new_state)
 
+    @skipUnlessDBFeature("allows_multiple_constraints_on_same_fields")
```

Comment:
I added `skipUnlessDBFeature` as I realized it won't work on Oracle.

Reply:
Good catch, thanks 🙏